theano/sparse/basic.py

```diff
@@ -1219,15 +1219,22 @@ def make_node(self, x, index):
                 # in case of slice is written in theano variable
                 start = ind.start
                 stop = ind.stop
-                if ind.step is not None:
-                    raise ValueError((
-                        "Using a slice with non-default step when "
-                        "indexing into a sparse matrix is not supported. "),
-                        ind, ind.step)
-
-                # If start or stop are None, make them a Generic constant
-                # Else, they should be converted to Tensor Variables of
-                # dimension 1 and int/uint dtype.
+                step = ind.step
+                # If start or stop or step are None, make them a Generic 
+                # constant. Else, they should be converted to Tensor Variables
+                # of dimension 1 and int/uint dtype.
+                if ind.step is  None or ind.step == 1:
+                    step = generic_None
+                else:
+                    if not isinstance(step, gof.Variable):
```

Comment:
Travis use scipy 0.11. This version raise an error is step isn't 1. We know that older version was just ignoring the step when it wasn't None. Just to help the user, can you assert here that scipy version is >= 0.11 ? That way we will be sure one error will be raised to the user (Theano one or scipy one) without asking us to test all scipy version.